api_tests/nodes/views/test_node_list.py

```diff
@@ -632,6 +632,28 @@ def test_creates_project_creates_project_and_sanitizes_html(self):
         assert_equal(res.json['data']['attributes']['description'], strip_html(description))
         assert_equal(res.json['data']['attributes']['category'], self.category)
 
+    def test_create_component_inherit_contributors(self):
+        parent_project = ProjectFactory(creator=self.user_one)
+        parent_project.add_contributor(self.user_two, permissions=[permissions.READ], save=True)
+        url = '/{}nodes/{}/children/{}'.format(API_BASE, parent_project._id, '?inherit_contributors=true')
```

Comment:
Very minor: no need to str.format with a string literal (`'?inherit_contributors=true'`). Ain't no thang, though